src/app/components/component-list/components-list-router.module.ts

```diff
@@ -14,9 +14,9 @@ const routes: Route[] = [
                 path: ':id',
                 component: ComponentViewerComponent,
                 children: [
-                    {path: 'api', component: ComponentApiComponent, pathMatch: 'full'},
-                    {path: 'examples', component: ComponentExamplesComponent, pathMatch: 'full'},
-                    {path: '**', redirectTo: 'api'}
+                    {path: 'API', component: ComponentApiComponent, pathMatch: 'full'},
```

Comment:
This is a bit inconsistent with our urls, so if we want to change the case on the tabs we could handle it by having a "display" version or by using pipe

Reply:
If you want, could you revert this change to the urls and I'll make the change so the tabs show up with the correct case. I'll approve

Reply:
Okay - cool, will do.  So yeah the display versions will be "API", "Examples", and "Usage"